Fix PivotGrid.onCellPrepared docissues (#1778)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxPivotGrid/1 Configuration/onCellPrepared.md

The position of a cell's row.

---

> This function allows you to customize cells and modify their content. The most usual use-cases are the following:

Reviewer: ```suggestion
This function allows you to customize cells and modify their content. Common use-cases are as follows:
```
